rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/clearingrequest/Sw360ClearingRequestService.java

```diff
@@ -130,4 +131,19 @@ public void updateClearingRequestForChangeInClearingSize(String crId, ClearingRe
             log.error("Error updating clearing request for change in clearing size: " + e.getMessage());
         }
     }
+
+    public String getUserNameByEmail(String userEmail) throws TException {
+        try {
+            User sw360User = getThriftUserClient().getByEmail(userEmail);
+            return sw360User.getFullname();
+        } catch (TException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private UserService.Iface getThriftUserClient() throws TTransportException {
+        THttpClient thriftClient = new THttpClient(thriftServerUrl + "/users/thrift");
+        TProtocol protocol = new TCompactProtocol(thriftClient);
+        return new UserService.Client(protocol);
+    }
```

Comment:
Please make use of `org.eclipse.sw360.datahandler.thrift.ThriftClients#makeUserClient()`

Reply:
done.